app/bundles/EmailBundle/Entity/StatRepository.php

```diff
@@ -658,19 +658,20 @@ public function checkContactsSentEmail($contacts, $emailId)
     }
 
     /**
-     * @param $emailId
+     * @param $contacts
+     * @param $emailIds
      *
      * @return array Formatted as [contactId => sentCount]
      */
-    public function getSentCountForContacts(array $contacts, $emailId)
+    public function getSentCountForContacts(array $contacts, array $emailIds)
```

Comment:
This is a backward compatibility change. If a plugin use this method then it will cause an error when the second param was changed to an array from an int. I'd normally ask you to create a new method and mark this one as deprecated as I can see it's used only in one place. But since this will be releases in a major version we can introduce this BC change. However, it must be documented in https://github.com/mautic/mautic/blob/5.x/UPGRADE-5.0.md. Please do so.